frontend/src/Components/StartPage.js

```diff
@@ -0,0 +1,312 @@
+import React from "react";
+import "../App.css";
+import "../CSS/graph.css";
+import Graph from "./Graph";
+import Table from "./Table";
+import Modal from "react-awesome-modal";
+import momemt from "moment";
+import * as moment from "moment";
+
+const AllDateOption = "ALL";
+const YearDateOption = "YEAR";
+const MonthDateOption = "MONTH";
+const DayDateOption = "DAY";
```

Comment:
isnt all of this belong in the graph component?

Reply:
But it set state in main component...?